Vagrantfile

```diff
@@ -6,8 +6,8 @@ Vagrant.configure("2") do |config|
     config.vm.box = "minimal/trusty64"
 
     # Create forwarding ports for client-guest machine access via localhost.
-    config.vm.network :forwarded_port, guest: 80, host: 8080
-    config.vm.network :forwarded_port, guest: 6000, host: 6000
+    config.vm.network :forwarded_port, guest: 80, host: 8080, auto_correct: true
```

Comment:
It will auto correct the ports so that there is no port conflicts. On the other hand you loose control over which port you can set. Also I am not sure yet how to access the port once it has been auto set. There must be some command to run in order to see what port has been forwarded.